### ISSUE TITLE
[videocenter] Video Center View All videos for a channel does not reflect the total amount of videos

### ISSUE THREAD

**Issue by AndyReiter** [craftercms/craftercms#5263]
#### VC BP Versions
* 1.1.41
* 2.0.22

#### Describe the bug
When selecting a channel and selecting the "View All" option, the resulting view does not reflect the total amount of videos. 

#### To Reproduce
1. Create a site based on the video-center blueprint
2. Navigate to Components --> Channels --> right-click and select "New Content". Select "Component - Channel"
3. Fill out the required information
4. Change the Max Videos to Display to be 12
5. Save & Close
6. Navigate to Taxonomies
7. Right-Click Channels under the Taxonomies/Channels and edit
8. Add a new item for the newly created channel and Save & Close
9. Navigate to videos and Right Click edit 12 videos and add them to the newly created channel
10. Once you have added 12 videos to the channel navigate back to the home page
11. Click Channels
12. Click the newly created Channel under "All Channels"
13. Click "View All" and notice that there are not 12 videos listed under the channel

#### Logs
N/A

#### Screenshots
N/A

**Comment by AndyReiter** [comment]
Still able to reproduce this issue in latest build:

```Studio Version Number:
4.0.0-SNAPSHOT-9c5b68

Build Number:
9c5b6886826333b3a8ec5de29c824ebdd0b4d440

Build Date:
4/14/2022, 10:34 AM

**Comment by sumerjabri** [comment]
@jvega190 the issue is in two places:
1. The Form says `Max Videos to Display`, please change that to `Max Featured Videos to Display`
2. The View All option in the rendering shows 10 videos, please add pagination

**Comment by sumerjabri** [comment]
@rart please finish reviewing the PR so we can merge and validate for v4.

**Comment by sumerjabri** [comment]
Publish the BP

**Comment by AndyReiter** [comment]
Confirmed Fixed in 4.0-SNAPSHOT.
